src/cli/index.ts

```diff
@@ -146,6 +146,26 @@ export const runCli = async () => {
       });
 
       cliResults.packages = packages;
+
+      // Skip if noGit flag provided
+      if (!cliResults.flags.noGit) {
+        const { gitInit } = await inquirer.prompt<{ git: boolean }>({
+          name: "git",
+          type: "list",
+          message: "Initialize a new git repository?",
+          choices: [
+            { name: "Yes", value: true, short: "Yes" },
+            { name: "No", value: false, short: "No" },
+          ],
+          default: true,
+        });
+        if (gitInit) {
```

Comment:
Additionally, rather than presenting a list with a yes or no answer, a simple confirm type will make more sense here. [LINK TO INQUIRER DOCS](https://github.com/SBoudrias/Inquirer.js#confirm---type-confirm)

Reply:
@MWhite-22 very true! I will work on this!

Reply:
👋 apologies for low quality PR, I have fixed my linter on my Windows and updated with the requested changes.
Tested the download on myside (was getting incorrect results before as which I now know was because I didn't rebuild the app with latest changes)
Apologies again!